src/v2/components/ChannelContentsFilter/index.tsx

```diff
@@ -39,11 +40,24 @@ export const ChannelContentsFilter = ({ channel }) => {
 
   if (channel.counts.contents === 0) return null
 
+  const channelLabel =
+    channel.counts.channels === 1
+      ? '1 channel'
+      : `${channel.counts.channels} channels`
+  const blockLabel =
+    channel.counts.blocks === 1 ? '1 block' : `${channel.counts.blocks} blocks`
+
+  const countLabels = []
+  if (channel.counts.blocks > 0) countLabels.push(blockLabel)
+  if (channel.counts.channels > 0) countLabels.push(channelLabel)
```

Comment:
I'm the kinda person that would find something like

```javascript
const { counts } = channel;

const counts = toSentence([
  ...(counts.channels > 0 ? [`${counts.channels} channel${counts.channels === 1 ? '' : 's'}`] : []),
  ...(counts.blocks > 0 ? [`${counts.blocks} block${counts.blocks === 1 ? '' : 's'}`] : []),
]);
```

to be more legible.

Tho I'd probably just pull out some different helpers here and try to get rid of underscore.string

So this is fine.

Reply:
Yeah, I knew you'd have opinions on this.

I think your version is nicer and yeah I would like to avoid `toSentence` so let me try to rework.